### DISCUSSION TITLE
Integrating Imagesharp with toast ui image editor

### DISCUSSION THREAD

**Discussion by aloksharma1** [SixLabors/ImageSharp#1788] [Q&A] [answered]
Hello, 

I am thinking on the idea to use Imagesharp or similar solution to create a .net connector with [tui.image-editor ](https://github.com/nhn/tui.image-editor), my question here is does imagesharp provides the api's to make all the Features of editor backend compatible with a .net core 5+ mvc solution.
So basically what i am doing is instead of exporting canvas i want to use editor image coordinates to manipulate files on server, to keep the image quality optimal.

Thanks for the help

**Comment by antonfirsov** [comment, marked as answer]
@aloksharma1 we have a lot of image effects/filters, you can browse them here:
https://github.com/SixLabors/ImageSharp/tree/master/src/ImageSharp/Processing/Extensions

Watch out for terminology, [Filters](https://github.com/SixLabors/ImageSharp/tree/master/src/ImageSharp/Processing/Extensions/Filters) mean color filters, you want to also look at [Dithering](https://github.com/SixLabors/ImageSharp/tree/master/src/ImageSharp/Processing/Extensions/Dithering), [Convolution](https://github.com/SixLabors/ImageSharp/tree/master/src/ImageSharp/Processing/Extensions/Convolution), [Effects](https://github.com/SixLabors/ImageSharp/tree/master/src/ImageSharp/Processing/Extensions/Effects) etc..

**Comment by aloksharma1** [comment]
thanks i will check thee in details, do you want me to close this thread (if i get all i want from these docs) or can i keep this open for future questions relating to this same issue.

**Reply by antonfirsov** [reply]
If you think my post above answers your main question press *Mark as answer*, to help guiding other users who find this discussion through search engines. This won't prevent you from asking other details under the discussion later, we get notifications for comments in the repo, and will likely come back.
